core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -4860,6 +4860,27 @@ public void putGeoHash(int index, long value) {
             notNull(index);
         }
 
+        public void putGeoHash(int index, int storageTag, long value) {
```

Comment:
Get/put API should be aligned in a sense that we don’t have getGeohash() method

Reply:
Not sure I follow, the `TableReader` provides:

```
reader.getCursor().getRecord().getGeoHashByte()
reader.getCursor().getRecord().getGeoHashShort()
reader.getCursor().getRecord().getGeoHashInt()
reader.getCursor().getRecord().getGeoHashLong()
```

on the other hand, `TableWriter` provides instance class `Row` with only put methods.

Reply:
nudge

Reply:
I would suggest that we introduce:

```
   row.putGeoHashByte();
   row.putGeoHashShort()
   row.putGeoHashInt()
   row.putGeoHashLong()
```

I would suggest to do the correct typing in reader thread so that writer thread does not have to perform nested `switch`

Reply:
yup, I did not think of that, thank you